src/libm/CMakeLists.txt

```
@@ -359,7 +359,12 @@ endif()
 # Original sleef sources
 set(STANDARD_SOURCES rempitab.c)
 
-add_library(${TARGET_LIBSLEEF} ${STANDARD_SOURCES})
+if(_INTERNAL_SLEEF_BUILD_SHARED_LIBS)
+  add_library(${TARGET_LIBSLEEF} SHARED ${STANDARD_SOURCES})
+else()
+  add_library(${TARGET_LIBSLEEF} STATIC ${STANDARD_SOURCES})
+endif()
```

```suggestion
add_library(${TARGET_LIBSLEEF} ${STANDARD_SOURCES})
```
I'm going to stop highlighting all the places this needs to be reverted.